### DISCUSSION TITLE
May 2023

### DISCUSSION THREAD

**Discussion by andrew** [octobox/octobox#3329] [Announcements]
## What's Changed
* Bump docker/build-push-action from 2.5.0 to 4.0.0 by @dependabot in https://github.com/octobox/octobox/pull/3315
* Bump sidekiq from 7.0.8 to 7.0.9 by @dependabot in https://github.com/octobox/octobox/pull/3316
* Bump pg from 1.4.6 to 1.5.1 by @dependabot in https://github.com/octobox/octobox/pull/3318
* Bump rack from 2.2.6.4 to 2.2.7 by @dependabot in https://github.com/octobox/octobox/pull/3319
* Bump i18n from 1.12.0 to 1.13.0 by @dependabot in https://github.com/octobox/octobox/pull/3320
* Bump pg from 1.5.1 to 1.5.2 by @dependabot in https://github.com/octobox/octobox/pull/3321
* Bump zeitwerk from 2.6.7 to 2.6.8 by @dependabot in https://github.com/octobox/octobox/pull/3322
* Bump pg from 1.5.2 to 1.5.3 by @dependabot in https://github.com/octobox/octobox/pull/3324
* Bump better_errors from 2.9.1 to 2.10.0 by @dependabot in https://github.com/octobox/octobox/pull/3323
* Bump mini_portile2 from 2.8.1 to 2.8.2 by @dependabot in https://github.com/octobox/octobox/pull/3326
* Bump sidekiq from 7.0.9 to 7.1.0 by @dependabot in https://github.com/octobox/octobox/pull/3327

**Full Changelog**: https://github.com/octobox/octobox/compare/april-2023...may-2023

<hr /><em>This discussion was created from the release <a href='https://github.com/octobox/octobox/releases/tag/may-2023'>May 2023</a>.</em>

**Comment by bertie** [comment]
test comment to see if it triggers a notification
